client/src/main.rs

```diff
@@ -1035,6 +1037,52 @@ fn override_endpoint(
     Ok(())
 }
 
+fn prompt_override_endpoint(
+    info: &Info,
```

Comment:
Super nit: `server_info` for the variable name? Or maybe even for the type name if you feel like it.